activerecord/lib/active_record/relation.rb

```diff
@@ -236,8 +236,8 @@ def find_or_initialize_by(attributes, &block)
     #
     # Please see further details in the
     # {Active Record Query Interface guide}[https://guides.rubyonrails.org/active_record_querying.html#running-explain].
-    def explain
-      exec_explain(collecting_queries_for_explain { exec_queries })
+    def explain(format = :text)
```

Comment:
We should also document this new option.

Reply:
I have added parameters section with some basic text. Feel free to suggest some better text since I'm not english native speaker.